great_expectations/experimental/datasources/pandas_datasource.py

```diff
@@ -233,19 +39,41 @@ def get_batch_list_from_batch_request(
     ),
 )
 
+
 CSVAsset = _ASSET_MODELS["csv"]
 ExcelAsset = _ASSET_MODELS["excel"]
 JSONAsset = _ASSET_MODELS["json"]
 ParquetAsset = _ASSET_MODELS["parquet"]
 
 
+def build_filesystem_data_asset_class(
```

Comment:
@Kilo59 Per your suggestion during our conversation, I was able to avoid the need for this method with everything working properly by having the interface implementations included as part of the model generation.  Thanks!